code/game/objects/items/devices/radio/encryptionkey.dm

```diff
@@ -10,6 +10,8 @@
 	var/syndie = FALSE
 	/// If true, the radio can say/hear on the special CentCom channel.
 	var/independent = FALSE
+	/// Determines if the key can be removed from a radio/headset
+	var/canRemove = TRUE
```

Comment:
`snake_case`

Reply:
hmm, not sure if that means you liked it, tagged someone or want something changed